src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnToReturnViewModelMap.cs

```diff
@@ -14,6 +14,7 @@ public class ReturnToReturnViewModelMap : IMap<ReturnData, ReturnViewModel>
         public decimal? NonObligatedTonnageTotalDcf = null;
 
         public List<AatfObligatedData> AatfObligatedData = new List<AatfObligatedData>();
+        private TonnageUtilities tonnageUtilities = new TonnageUtilities();
```

Comment:
This should be injected in the constructor, public ReturnToReturnViewModelMap(ITonnageUtilities tonnageUtilities)

Reply:
Done